src/Components/App/App.js

```diff
@@ -11,6 +11,15 @@ import PageNotFound from '../PageNotFound/PageNotFound'
 import './App.scss'
 
 function App() {
+  if (!('Notification' in window)) {
```

Comment:
I think it does make sense to leave this in App for the time being, but I'm curious what best practices are

Reply:
I would agree, it's nice to finally start messing with some of these other things in a React app, I'm glad this stretch is forcing us to pick some of this stuff apart.